homeassistant/components/onvif/camera.py

```diff
@@ -192,8 +192,26 @@ def __init__(self, hass, config):
         try:
             system_date = dt_util.utcnow()
             device_time = await devicemgmt.GetSystemDateAndTime()
-            if device_time:
-                cdate = device_time.UTCDateTime
+            if not device_time:
+                _LOGGER.debug(
+                    """Couldn't get camera '%s' date/time.
+                    GetSystemDateAndTime() return null/empty""",
+                    self._name,
+                )
+                return
+
+            tzone, cdate = (
+                (dt_util.UTC, device_time.UTCDateTime)
+                if device_time.UTCDateTime
```

Comment:
We should not use multiline inline if-statements. They are unreadable.

Reply:
Done